src/plugin/ogc/wfs/wfslayerconfig.js

```diff
@@ -333,11 +333,15 @@ class WFSLayerConfig extends AbstractDataSourceLayerConfig {
       new SemiMinorMapping()
     ];
 
+    const hasEllipse = options['mappings'].some((temp) => this.isEllipticalMapping(temp));
```

Comment:
We should probably sanity check that `options['mappings']` is actually defined. Our WFS descriptor always sets it, but this layer config could be used elsewhere without a similar change.